### PR TITLE
Imperia charcoal kiln added

### PR DESCRIPTION
**Pull request by Cheese-wood1**

### CODE REVIEW

File: common/production_method_groups/imperia_industry.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿pmg_charcoal_kiln = {
+	texture = "gfx/interface/icons/generic_icons/mixed_icon_base.dds"
```
Comment (HiddeLekanne) [inline common/production_method_groups/imperia_industry.txt:2]: needs proper icon

File: common/production_methods/imperia_industry.txt [hunk 2]
```diff
@@ -22,3 +21,134 @@
 		}
 	}
 }
+
+pm_basic_charcoal_production = {
+	texture = "gfx/interface/icons/production_method_icons/picks_and_shovels.dds"
```
Comment (HiddeLekanne) [inline common/production_methods/imperia_industry.txt]: needs proper icon

File: common/production_methods/imperia_industry.txt [hunk 3]
```diff
@@ -22,3 +21,134 @@
...
+			#Productivity with merchant guilds: 6.24 -> ~8% more efficient than basic pm coal mine
+		}
+		level_scaled = {
+			building_employment_laborers_add = 4500
+		}
+	}
+}
+
+pm_improved_charcoal_production = {
+	texture = "gfx/interface/icons/production_method_icons/picks_and_shovels.dds"
```
Comment (HiddeLekanne) [inline common/production_methods/imperia_industry.txt]: needs proper icon

File: common/production_methods/imperia_industry.txt [hunk 4]
```diff
@@ -22,3 +21,134 @@
 		}
 	}
 }
+
+pm_basic_charcoal_production = {
```
Comment (HiddeLekanne) [inline common/production_methods/imperia_industry.txt:25]: Needs pollution

File: common/production_methods/imperia_industry.txt [hunk 5]
```diff
@@ -22,3 +21,134 @@
...
+			goods_output_coal_add = 30
+			#Productivity with merchant guilds: 6.24 -> ~8% more efficient than basic pm coal mine
+		}
+		level_scaled = {
+			building_employment_laborers_add = 4500
+		}
+	}
+}
+
+pm_improved_charcoal_production = {
```
Comment (HiddeLekanne) [inline common/production_methods/imperia_industry.txt:46]: Needs pollution

File: common/building_groups/00_building_groups.txt [hunk 6]
```diff
@@ -93,6 +93,15 @@ bg_munition_plants = {
 	infrastructure_usage_per_level = 3
 }
 
+bg_charcoal_kiln = {
+	parent_group = bg_light_industry
+
+	urbanization = 5
+	infrastructure_usage_per_level = 1
```
Comment (HiddeLekanne) [inline common/building_groups/00_building_groups.txt]: 10 urbanization and 2 infrastructure usage would be more consistent with the building costs 400 (similar to a mine)

**Review by HiddeLekanne — CHANGES_REQUESTED** [review]

File: common/production_methods/imperia_industry.txt (hunk 3 above)
Comment (Danarca) [reply common/production_methods/imperia_industry.txt]: charcoal_icons
Had some trouble finding suitable icons, but these should evocate the feeling well enough.

File: common/production_method_groups/imperia_industry.txt (hunk 1 above)
Comment (Danarca) [reply common/production_method_groups/imperia_industry.txt:2]: This is actually correct; this is the icon that shows up when you have mixed PMs activated.

**Review by HiddeLekanne — APPROVED** [review]

**Pull request merged**